t/rt75353-innodb-lock-timeout.t

```diff
@@ -18,7 +18,7 @@ if (!@ilwtenabled) {
 }
 
 my $have_innodb = 0;
-if (!MinimumVersion($dbh1, '5.6')) {
+if (!MinimumVersion($dbh1, '4.1.2')) {
```

Comment:
Above test is for usage of SHOW ENGINES, have_innodb is fallback.

And I personally do not think that it is possible to use current DBD::mysql with older libmysqlclient as 4.1. Our Travis CI tests cover MySQL 4.1 as the oldest version.